README.md

```diff
@@ -6,4 +6,28 @@
 ## About
 * Its used by "Registration Client" and "Resident Service" to create packets.
 * Its used by "Registration Processor" to read packet.
-  
+
+## Overview
+
+* This repository contains the source code MOSIP Packet Manager module.  For an overview refer [here](https://docs.mosip.io/1.2.0/modules/packet-manager). This Module Exposes some API endpoints that we can explore in this [Swagger](https://api-internal.dev.mosip.net/commons/v1/packetmanager/swagger-ui/index.html?configUrl=/commons/v1/packetmanager/v3/api-docs/swagger-config#/).  
+---
+## Build & run (for developers)
+The project requires JDK 21.0.3  and mvn version 3.9.6
+
+1. Build and install:
+   $ cd commons-packet <br>$ mvn clean install -Dgpg.skip=true
+___
+## Build Docker for a service
+>$ cd `<service folder>` <br>
+$ docker build -f Dockerfile
+# Property file used by the application
```

Comment:
Use this heading as "Configuration" as we are following the same at other places. Check the text size as well. It is larger than other texts. It should be same.

Reply:
Done.